shared/chat/conversation/input-area/normal/platform-input.desktop.js

```diff
@@ -215,143 +210,154 @@ class PlatformInput extends Component<PlatformInputProps & OverlayParentProps, S
-              onSelectChannel={this.props.switchChannelMention}
-              filter={this.props.channelMentionFilter}
-            />
-          )}
-          <Box style={{...globalStyles.flexBoxRow, alignItems: 'flex-end'}}>
-            <input
-              type="file"
-              style={{display: 'none'}}
-              ref={this._filePickerSetRef}
-              onChange={this._pickFile}
-              multiple={true}
-            />
-            <Input
-              className={'mousetrap' /* className needed so key handler doesn't ignore hotkeys */}
-              autoFocus={false}
-              small={true}
-              style={styleInput}
-              ref={this._inputSetRef}
-              hintText={hintText}
-              hideUnderline={true}
-              onChangeText={this._onChangeText}
-              uncontrolled={true}
-              multiline={true}
-              rowsMin={1}
-              rowsMax={5}
-              onKeyDown={this._onKeyDown}
-              onEnterKeyDown={this._onEnterKeyDown}
-            />
-            {flags.explodingMessagesEnabled &&
-              this.props.isExploding &&
-              !this.props.isEditing &&
-              !this.state.hasText && (
-                <Icon
-                  color={globalColors.black_20}
-                  fontSize={34}
-                  hoverColor={globalColors.black_20}
-                  onClick={this._inputFocus}
-                  style={styleBoomIcon}
-                  type="iconfont-boom"
-                />
-              )}
-            {flags.explodingMessagesEnabled &&
-              this.props.showingMenu && (
-                <SetExplodingMessagePopup
-                  attachTo={this.props.attachmentRef}
-                  conversationIDKey={this.props.conversationIDKey}
-                  onAfterSelect={this._inputFocus}
-                  onHidden={this.props.toggleShowingMenu}
-                  visible={this.props.showingMenu}
+        {this.props.channelMentionPopupOpen && <MentionCatcher onClick={this._channelMentionCatcherClick} />}
+        {this.props.channelMentionPopupOpen && (
+          <ChannelMentionHud
+            conversationIDKey={this.props.conversationIDKey}
+            selectDownCounter={this.props.downArrowCounter}
+            selectUpCounter={this.props.upArrowCounter}
+            pickSelectedChannelCounter={this.props.pickSelectedCounter}
+            onPickChannel={this.props.insertChannelMention}
+            onSelectChannel={this.props.switchChannelMention}
+            filter={this.props.channelMentionFilter}
+          />
+        )}
+        <Kb.Box
+          style={collapseStyles([
+            styles.inputWrapper,
+            {
+              backgroundColor: this.props.isEditing ? globalColors.yellow3 : globalColors.white,
+              borderColor: this.props.explodingModeSeconds
+                ? globalColors.black_75
+                : this.props.isEditing
```

Comment:
don't think this second ternary is needed

Reply:
It was left over from testing colors. Thanks!